packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -663,7 +667,7 @@ function isOverTarget(point: EventPoint, target: HTMLElement) {
 
 function shouldPreventDefault(target: Element) {
   // We cannot prevent default if the target is inside a draggable element.
-  return !target.closest('[draggable="true"]');
+  return !(target.closest('[draggable="true"]') || ((target.role === 'button' || target.tagName === 'BUTTON') && target.type === 'submit'));
```

Comment:
I don't think this check is quite right, also, it has a TS error due to Element not having role or type defined on it